keyboards/hfdkb/keyboard_sw/mk47/rules.mk

```
@@ -0,0 +1,17 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic
```

This comment is out of sync, I think.